python/lib/sift_py/ingestion/config/telemetry.py

```diff
@@ -5,12 +5,12 @@
 
 from typing_extensions import Self
 
+from sift_py._internal.channel import channel_fqn as _channel_fqn
```

Comment:
This doesn't need to be aliased. Could you update everything to just be `channel_fqn`?